tests/testdata/certs/default/cert-chain.pem

```diff
@@ -1,19 +1,20 @@
 -----BEGIN CERTIFICATE-----
-MIIDJzCCAg+gAwIBAgIJAKTFsLiNOkDCMA0GCSqGSIb3DQEBCwUAMBgxFjAUBgNV
-BAMMDWNsdXN0ZXIubG9jYWwwIBcNMjAwMzEyMTU1OTAxWhgPMjEyMDAyMTcxNTU5
-MDFaMBgxFjAUBgNVBAMMDWNsdXN0ZXIubG9jYWwwggEiMA0GCSqGSIb3DQEBAQUA
-A4IBDwAwggEKAoIBAQDjLFOFDcaaKPORom0CkmHcKYtRRFnda+CnSYdOKoF1Zbvt
-QBXZqXzpQM4UlrujR1UG6HudRYWL3bvhIki8hV0F6iMhyq14FMPT9sPyEuVbxAA5
-n/1GgujtHjIEf4R0F+n8KP4IzMjFWgTfO82X568wuGz3oL2JOqSa81zOiEFv2mHV
-4r0auwlMYsPiOTeLOYYVZGaQ/swOgpR40CgnkoYphJKcEgfJ5YcAvbw7HHWBQXpZ
-KA44Ii8IUasORC6/VotMzxTXvMNokMACLQBtwmftCfvz0V2LPoJ83bK/rJ84t4LU
-Vd2whWSI8iQpPlEpz3soYliptO2Y/W3TssMKIExLAgMBAAGjcjBwMAkGA1UdEwQC
-MAAwCwYDVR0PBAQDAgXgMB0GA1UdJQQWMBQGCCsGAQUFBwMCBggrBgEFBQcDATA3
-BgNVHREEMDAuhixzcGlmZmU6Ly9jbHVzdGVyLmxvY2FsL25zL2RlZmF1bHQvc2Ev
-ZGVmYXVsdDANBgkqhkiG9w0BAQsFAAOCAQEA1Glwoy03FBk36RmadFj6REJ6CR+O
-tJqqUPu6JLeT2ohKUZdC7Ns/CHm8c74rI0HuzDOIgmCjqQ3XY3MeGibp6zERzKFA
-2t7LPRzzRO2TFOkPC8pxUCfIqQ/WYjfL2HRsN//YoJa2BzrMHAah3HimQFTS+Kj0
-6iuKHnmBktrVgl/mdcsAU9cCPIfEg3aJBfQCbuAFVxIf5IXD7z65xiR/fpSh1EQz
-dZbKy8+GLzjUqi3uxFkxNqSPhLuA9K2QzZCp1C206TGQ7GR8CFcMLPYTAtV/TgPB
-9RSmjRU7ihdBinKfF7BXW3XkQ2mICngQFW51MKm4Pys3r7VDRy+dRl8/dg==
+MIIDMjCCAhqgAwIBAgIUaAVaH1ssljSp51Hm0bAEJUVf4dMwDQYJKoZIhvcNAQEL
```

Comment:
Needed to recreate the certs to add `istiod` as part of the SAN, before it was only pilot. I also included the generate.sh script that was used to create that, before it was ad hoc